Readability.js

```diff
@@ -1723,7 +1825,7 @@ Readability.prototype = {
         // ominous signs, remove the element.
         var p = node.getElementsByTagName("p").length;
         var img = node.getElementsByTagName("img").length;
-        var li = node.getElementsByTagName("li").length-100;
+        var li = node.getElementsByTagName("li").length - 100;
```

Comment:
Let's add new eslint rules to avoid the bad code style like this. Already filed a follow up [issue][issue].

[issue]: https://github.com/mozilla/readability/issues/351